content/docs/command-reference/plots/index.md

```diff
@@ -98,72 +100,83 @@ names in the `train` array below:
 
 ## Plot templates (data series only)
 
-Users have the ability to change the way data-series plots are displayed by
-modifying the [Vega-Lite specification](https://vega.github.io/vega-lite/), thus
-generating plots in the style that best fits the their needs. This keeps
-<abbr>DVC projects</abbr> programming language agnostic, as it's independent
-from user display configuration and visualization code.
+DVC uses [Vega-Lite](https://vega.github.io/vega-lite/) JSON specifications to
+create visualizations out of users' data. Pre-defined specifications are called
+templates.
```

Comment:
Repeat of https://github.com/iterative/dvc.org/pull/3697#discussion_r910466305 + https://github.com/iterative/dvc.org/pull/3697#pullrequestreview-1022147287
```suggestion
DVC uses [Vega-Lite](https://vega.github.io/vega-lite/) JSON specifications
to create plots from user data. A set of built-in _plot templates_ are included.
```